notion_client/client.py

```diff
@@ -120,7 +120,7 @@ def _parse_response(self, response: Response) -> Any:
             try:
                 body = error.response.json()
                 code = body.get("code")
-            except json.JSONDecodeError:
+            except json.JSONDecodeError:  # pragma: no cover
```

Comment:
Why not testing this line?

Reply:
Because I can't generate a request from the Notion API with uncorrect JSON data to test. Help is welcomed with this specific case

Reply:
Can't we just test it with http://httpstat.us/200? It doesn't return JSON.

Reply:
Nevermind, it's being covered, I removed the # no cover comment :)